### ISSUE TITLE
Vite built-in env inconsistent with typing

### ISSUE THREAD

**Issue by melishev** [garronej/vite-envs#26]
In the vite-inv.d.ts file, I get content like this:

```ts
/// <reference types="vite-envs/client" />

type ImportMetaEnv = {
  // Auto-generated by `npx vite-envs update-types` and hot-reloaded by the `vite-env` plugin
  // You probably want to add `/src/vite-env.d.ts` to your .prettierignore
  BASE_URL: string
  MODE: string
  DEV: boolean
  PROD: boolean
```

The problem is that when calling import.meta.env.PROD I will get a string, which seems to be normal behavior for an environment variable, so why the confusion?

**Comment by garronej** [comment]
Hello @melishev,  

This is a bug. Thank you for reporting it. I'll release a new version with a fix

**Comment by garronej** [comment]
It's fixed in `4.3.3`.  
The runtime values you're getting matches the type definitions.  

Aditionally this release makes vite-envs self contained. ([It won't add any thing to your node module but itself](https://www.npmjs.com/package/vite-envs?activeTab=dependencies))
